package.json

```diff
@@ -12,7 +12,7 @@
   "files": [
     "dist"
   ],
-  "main": "dist/index.js",
+  "private": true,
```

Comment:
Why is this changing? We should indicate to npm that main is dist/index.js, yes?

Reply:
Those two changes were requested by webpack.  I think it's to avoid accidental commits?  I expected that we would have to add it back at some point.  

https://webpack.js.org/guides/getting-started

Reply:
>We also need to adjust our package.json file in order to make sure we mark our package as private, as well as removing the main entry. This is to prevent an accidental publish of your code.

 In our case this is actually not a private module, so we should keep the main entry. Otherwise it will not work from NPM install.